packages/fast-components-react-base/src/slider/slider.spec.tsx

```diff
@@ -673,6 +673,21 @@ describe("Slider", (): void => {
         document.body.removeChild(container);
     });
 
+    test("constrainToStep rounds fractions of a step properly", (): void => {
+        const rendered: any = mount(
+            <Slider
+                range={{
+                    minValue: 0,
+                    maxValue: 100,
```

Comment:
you didn't supply a step here?

Reply:
The function we are testing is called directly and takes step as a second parameter. 
"expect(rendered.instance()["constrainToStep"](16, 10)).toBe(20);"

Reply:
I see, I expected it to be part of the range property like it is in JSON schema - https://json-schema.org/understanding-json-schema/reference/numeric.html#multiples and since step is an attribute added to the input along with min/max this test seemed unintuitive from a prop perspective

Reply:
I will add a note identifying the params on the function test